webapp/components/Notification/Notification.spec.js

```diff
@@ -198,4 +199,71 @@ describe('Notification', () => {
       })
     })
   })
+
+  describe('given a notification about the user has filed a report', () => {
+    beforeEach(() => {
+      propsData.notification = {
+        reason: 'filed_report_on_resource',
+        from: {
+          __typename: 'Report',
+          id: 'reportOnUser',
+          filed: [
+            {
+              reasonCategory: 'discrimination_etc',
+              reasonDescription: 'This user is harassing me with bigoted remarks!',
+              reportedResource: {
+                __typename: 'User',
+                id: 'badWomen',
+                slug: 'mrs.-badwomen',
+                name: 'Mrs. Badwomen',
+              },
+            },
+          ],
+        },
+      }
+    })
+
+    it('renders reason', () => {
+      wrapper = Wrapper()
+      expect(wrapper.find('.reason-text-for-test').text()).toEqual(
```

Comment:
same as :point_up: ... if you added this class, please use `data-test="reason-text"` as we decided we don't want classes for testing purposes.
If a class exists and can be used to select it, we should use it, otherwise, we should add `data-test`
If you didn't add this, could you please refactor it?

Reply:
Done dear 🤗